bin/registerServerTopology-func.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * Copyright 2005 - 2020 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+/**
+ * Ask question. The echo of keyboard can be disabled
+*
+* @param string $question
+* @param boolean $hidden
+* @return string
+*/
+function askQuestion(string $question, $hidden = false): string
+{
+    if ($hidden) {
+        system("stty -echo");
+    }
+    printf("%s", $question);
+    $handle = fopen("php://stdin", "r");
+    $response = trim(fgets($handle));
+    fclose($handle);
+    if ($hidden) {
+        system("stty echo");
+    }
+    printf("\n");
+    return $response;
+}
+
+/**
+ * Format the response for API Request
+ *
+ * @param integer $code
+ * @param string $message
+ * @param string $type
+ * @return string
+ */
+function formatResponseMessage(int $code, string $message, string $type = 'success'): string
+{
+    switch ($type) {
+        case 'error':
```

Comment:
As success and default will trigger the same message, shouldn't the error and success cases been inverted ?